src/main/java/ru/javawebinar/topjava/service/MealService.java

```diff
@@ -0,0 +1,15 @@
+package ru.javawebinar.topjava.service;
+
+import ru.javawebinar.topjava.model.Meal;
+
+import java.util.List;
+
+public interface MealService {
+    public void saveMeal(Meal meal);
+
+    public void removeMeal(Integer id);
+
+    public Meal getMealById(Integer id);
+
+    public List<Meal> listMeals();
```

Comment:
The names of the methods are predicate. So you should edit it

Reply:
Don't get it. Explane please.